english/staypuft/ghostbusters/where-is-my-proton-pack.md

```diff
@@ -24,4 +30,4 @@ After so much excitement you decide to:
 
 [Watch Ghostbusters and see if you're in the credits?](watch-ghostbusters/watch-ghostbusters.md)
 
-[Jump to another Bill Murray movie](billMurrayMovies/billMurrayMovies.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.